Sources/OpenAI/Private/StreamingSession.swift

```diff
@@ -60,14 +60,23 @@ extension StreamingSession {
         if stringContent.isEmpty {
             return
         }
-        let jsonObjects = "\(previousChunkBuffer)\(stringContent)"
+
+        // Join the previous chunk buffer with the new chunk, and filter out comments.
+        // FIX OpenRouter: https://github.com/tisfeng/Easydict/issues/743
+        let filteredContent = "\(previousChunkBuffer)\(stringContent)"
             .trimmingCharacters(in: .whitespacesAndNewlines)
+            .components(separatedBy: .newlines)
+            .filter { !$0.hasPrefix(":") }  // Filter out comments, like ": OPENROUTER PROCESSING"
```

Comment:
Hey, @tisfeng!
Is there any documentation on OpenRouter API about this line? We'd like to be sure we don't get a regression here and do not break any other providers.

Reply:
And what do you think about writing unit tests that cover both cases (with and without comments)